sympy/core/tests/test_wester.py

```diff
@@ -2093,9 +2093,7 @@ def test_T9():
     assert limit(hyper((1, k), (1,), z/k), k, oo) == exp(z)
 
 
-@XFAIL
 def test_T10():
-    # raises PoleError should return euler-mascheroni constant
     limit(zeta(x) - 1/(x - 1), x, 1)
```

Comment:
This test should be fixed instead, not UnXFAIL'ed.

BTW, here is a pretty-printing bug too:
```python
In [1]: limit(zeta(x) - 1/(x - 1), x, 1)
Out[1]: 
              1  
 lim ζ(x) - ─────
x─→1⁺       x - 1

In [2]: print _
Limit(zeta(x) - 1/(x - 1), x, 1)
```